tools/python/maps_generator/utils/file.py

```diff
@@ -48,6 +62,16 @@ def download_file(url: AnyStr, name: AnyStr, download_if_exists: bool = True):
         logger.info(f"File {name} already exists.")
         return
 
+    if is_file_uri(url):
+        #url uses 'file://' scheme
+        try:
+            copy_overwrite(file_uri_to_path(url), name)
+        except FileNotFoundError as e:
+            raise e
```

Comment:
Why do you need a try-except if an exception is propagated anyway? Let's remove it )